.travis.yml

```diff
@@ -37,6 +37,16 @@ matrix:
       os: linux
       env: TASK=verify-docs
       script: ./bin/verify-doc-changes.sh
+    - node_js: "10"
+      os: linux
+      env:
+        - TASK=test-repository-mongodb
+      services:
+        - mongodb
+      script:
+        - npm run postinstall -- --scope "@loopback/test-repository-mongodb" --include-filtered-dependencies --force-local
+        - npm run build -- --scope "@loopback/test-repository-mongodb" --include-filtered-dependencies
+        - cd acceptance/repository-mongodb && npm run mocha
```

Comment:
Note that I am intentionally leaving acceptance tests OUT OF `npm test` and configuring a new Travis CI job to run them in parallel. My intention is to add new scripts allowing developers to run these tests locally, but only when they decide to do so.

Reply:
That's good, but these tasks still run and are required for passing builds right?

Reply:
Yes, these tasks are executed and are required to consider a CI build as success.